packages/cli/bin/devrun

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+
+require('@oclif/command').run()
+.then(require('@oclif/command/flush'))
+.catch(err => console.log(err))
```

Comment:
Yeah, it seems to me that it could be useful in the future as well though there is probably a better way to do it. It's only a few bytes so I don't think there is any harm in leaving it here, at least until we have a way to turn on stack traces.